util/argo/argo.go

```diff
@@ -310,6 +316,17 @@ func ValidatePermissions(ctx context.Context, spec *argoappv1.ApplicationSpec, p
 	return conditions, nil
 }
 
+// APIGroupsToVersions converts list of API Groups into versions string list
+func APIGroupsToVersions(apiGroups []metav1.APIGroup) []string {
```

Comment:
Can we have a test for this method, please?

Reply:
I knew you would ask. Sorry for being lazy. Added.

Reply:
Thanks :+1: